RELEASE/scripts/autoscend/iotms/clan.ash

```diff
@@ -342,6 +342,20 @@ boolean drinkSpeakeasyDrink(item drink)
 	return cli_execute("drink 1 " + drink);
 }
 
+boolean drinkSpeakeasyDrink(string drink)
+{
+	if(!(auto_get_clan_lounge() contains $item[Clan Speakeasy]))
+	{
+		return false;
+	}
+
```

Comment:
These lines are redundant with the check that already happens in `drinkSpeakeasyDrink(item)`. We might as well remove them while we're moving stuff around.

Reply:
That should be a followup PR.
Due to its massive size I am trying to limit myself to actions that only need a validate as far as testing goes. That is, moving an entire function unchanged, reorganizing the header, and comments and the like